File: Lesson7/Car.java
// Java example of car class for your reference

public class Car {
    // instance variables
    public String make;
    public String model;
    public String wheels;
    public String color;
    
    // constructor
    Car(String make, String model, String wheels, String color){
        this.make = make;
        this.model = model;
        this.wheels = wheels;
        this.color = color;
    }


    //example of a method
    public void goBeep(){
        System.out.println("BEEP BEEP!!!");
    }

    //another public method
    public void getTurbo(){
        this.complexNo2Injection();
    }

    //private methods in java explicitly say private,
    //and java doesn't let users of the class call these functions directly 
    //to use whats in this function, you have to call "getTurbo" above
    private void complexNo2Injection(){
        //some complex distribution No2 injection into the engine....
        System.out.println("VROOM VROO!!!!");
    }

}

File: Lesson7/Car.py
#python example of a class for your reference

class Car:

    #constructor
    def __init__(make,model,wheels,color):
        self.make = make
        self.model = model
        self.wheels = wheels
        self.color = color

    #example of a public method
    def goBeep():
        print('BEEP BEEP!!!')

    
    def getTurbo():
        this.__complexNo2Injection()

    #python denotes private methods with a double-underscore
    #but just like javascript...this is just a visual indication that you might not want to mess with this.
    # and they are laxed enough to let you cal it.
    def __complexNo2Injection():
        print('BEEP BEEP!!!')

    
    